README.md

```diff
@@ -1,4 +1,32 @@
 # Kubernetes Metrics Server
 
-more details TBD
+## User documentation
 
+You can find the user guide in
+[the official Kubernetes documentation](https://kubernetes.io/docs/tasks/debug-application-cluster/core-metrics-pipeline/).
+
+## Design
+
+The project was designed in two phases:
+
+- [Metrics API](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/resource-metrics-api.md)
+- [Metrics Server](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/metrics-server.md)
+
+For the broader view of monitoring in Kubernetes take a look into
+[Monitoring architecture](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/monitoring_architecture.md)
+
+## Deployment
+
+Compatibility matrix:
+
+Metrics Server | Metrics API | supported Kubernetes version
+---------------|-------------|-----------------------------
+0.2 | v1beta1 | 1.8+
+0.1 | v1alpha1 | 1.7
+
+
+In order to deploy metrics-server in your cluster run the following command
+
+```bash
+kubectl create -f deploy/
```

Comment:
Then you should specify that you have to run this from this repo's directory

Reply:
done